### PR TITLE
Update users.json

### PR DESCRIPTION
**Pull request by lassv**
Added treeprofile.com

### CODE REVIEW

File: users.json [hunk 1]
```diff
@@ -288,5 +289,13 @@
     "infoLink": "https://flat.whiteboard.agora.io/en/",
     "pinned": true,
     "width": 120
+  },
+  
+   {
+    "caption": "Treeprofile",
+    "image": "https://www.treeprofile.com/assets/img/icon-green.png",
+    "infoLink": "https://www.treeprofile.com",
```
Comment (juanpicado) [inline users.json:297]: What is this exactly?

Comment (lassv) [reply users.json:297]: @juanpicado My little cool website, where users can have their own profiles. And we are using **pnpm** for the project

File: users.json [hunk 2]
```diff
@@ -288,5 +289,13 @@
...
+    "pinned": true,
```
Comment (juanpicado) [inline users.json:298]: This should be `false` by default, could you update it?

Comment (lassv) [reply users.json]: @juanpicado done

**Review by juanpicado — APPROVED** [review]
LGTM, thanks @lassv good luck with the initiative 💯

**Pull request merged**